website/guides/07_kube-prom-stack.mdx

```diff
@@ -0,0 +1,228 @@
+
+![stack-diagram](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/2fcquzgd73xf9cc83ap1.png)
+
+**Top Layer:**
+
+- **User:** Interacts with Grafana and Kubernetes API.
+
+**Visualization and Alerting Layer:**
+
+- **[Grafana](https://grafana.com/):** Connects to Prometheus for data visualization.
+- **[Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/):** Connected to Prometheus for alert management.
+- **[Prometheus Server](https://github.com/prometheus/prometheus):** Central component collecting and storing metrics.
+- **[ServiceMonitors & PodMonitors](https://observability.thomasriley.co.uk/prometheus/configuring-prometheus/using-service-monitors/):** Define how Prometheus scrapes metrics.
+- **[Prometheus Rules](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/):** Includes both alerting and recording rules.
+
+**Exporters Layer:**
+
+- **[Node Exporter](https://prometheus.io/docs/guides/node-exporter/):** Collects node-level metrics.
+- **[Kube State Metrics](https://github.com/kubernetes/kube-state-metrics):** Collects metrics from Kubernetes API objects.
+- **Other Exporters:** Additional exporters for various applications and services.
+
+**Kubernetes Cluster:**
+
+- **Kubernetes Nodes**: Running applications and system components.
+- **Applications:** Monitored by the Kube Prometheus Stack.
+
+Luckily, [Glasskube](https://github.com/glasskube/glasskube) now supports the `Kube-Prometheus-Stack`. Package configuration, lifecycle management, and installation can be done in record time.
+
+In this blog post, we will explore the steps to configure and install the `Kube-Prometheus-Stack` using [Glasskube](https://glasskube.dev/), wasting no unnecessary time wrestling with never-ending values files and getting you working dashboards and alerts quicker than ever before.
+
+### Requirements: 
+
+- Access to a **Kubernetes cluster** ([Minikube](https://minikube.sigs.k8s.io/docs/start) will be fine)
+- [Glasskube](https://github.com/glasskube/glasskube) installed
+- An extra screen for all the cool dashboards you are going to want to look at all the time. 🤣
+
+## Create a cluster
+
+Install [Minikube](https://minikube.sigs.k8s.io/docs/start) then run: 
+```
+minikube start
+```
+Check your installation by running:
+```
+minukube status
+```
+Desired output:
+```
+➜  ~ minikube status
+minikube
+type: Control Plane
+host: Running
+kubelet: Running
+apiserver: Running
+kubeconfig: Configured
+```
+## Install Glasskube
+
+If you already [installed](https://glasskube.dev/docs/getting-started/install/) glasskube you can skip this step.
+If not, glasskube can easily be installed by following your distribution's specific instructions.
+
+MacOs:
+```
+brew install glasskube/tap/glasskube
+```
+Linux (Ubuntu/Debian)
+```
+curl -LO https://releases.dl.glasskube.dev/glasskube_v0.4.0_amd64.deb
```

Comment:
`brew install` is also possible for linux, and is a bit nicer to read than curl + dpkg. I would either just directly link to the installation guides, or not differentiate between Mac and Linux here, and just have the brew line. If you want to keep it that way, please update the version number to the latest.

Reply:
I'll use the standardised installation steps used in the other docs